src/storage-client/src/storage_collections.rs

```diff
@@ -1316,11 +1336,12 @@ where
     // TODO(aljoscha): It would be swell if we could refactor this Leviathan of
     // a method/move individual parts to their own methods.
     #[instrument(level = "debug")]
-    async fn create_collections(
+    async fn create_collections_for_bootstrap(
         &self,
         storage_metadata: &StorageMetadata,
         register_ts: Option<Self::Timestamp>,
         mut collections: Vec<(GlobalId, CollectionDescription<Self::Timestamp>)>,
+        migrated_storage_collections: &BTreeSet<GlobalId>,
```

Comment:
I'm not a big fan of this change because it feels really invasive to change the name, but I also didn't want to have to pass in an empty set from all callers. Let me know if anyone has any opinions.

Reply:
I also don't like it, but AFAIK we'll roll this back again at some point in the future.